src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategy.kt

```diff
@@ -871,76 +871,43 @@ abstract class ManagedStrategy(
         return null
     }
 
+    /**
+     * This method is invoked by a test thread before each method invocation.
```

Comment:
This documentation (including the parameters) looks outdated

Reply:
I decided to remove this kdoc, because
1. It mostly duplicates kdoc of `Injections::beforeMethodCall`
2. There is no documentation for other methods of `ManagedStrategy` like `beforeReadField`. 
3. The documentation should be put in `EventTracker` interface anyway, because it is the base interface.